src/darwin/list.mm

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2017 resin.io
```

Comment:
Should this be `Copyright 2018 Robin Andersson` ? :man_shrugging:

Reply:
@lurch Had to read up about it, and since there is no CTA for this project I think the correct would be to have multiple Copyrights: https://opensource.stackexchange.com/questions/886/copyright-and-contributing-to-an-open-source-project

Not that picky about that.